user-interface/src/main/java/life/qbic/datamanager/views/projects/project/ontology/OntologyLookupComponent.java

```diff
@@ -159,13 +160,36 @@ private Span createHeader(String label, String curieText) {
       title.addClassName("ontology-item-title");
       Span curie = new Tag(curieText);
       curie.addClassNames("primary", "clickable");
+
+      Icon copyIcon = initCopyIcon();
+      Span header = new Span(title, curie, copyIcon);
+
       curie.addClickListener(
-          event -> UI.getCurrent().getPage().executeJs("window.copyToClipboard($0)", curieText));
-      Span header = new Span(title, curie, initCopyIcon(curieText));
+          event -> handleCopyClicked(header, curieText));
+      copyIcon.addClickListener(
+          event -> handleCopyClicked(header, curieText));
+
       header.addClassName("header");
       return header;
     }
 
+    private void handleCopyClicked(Span header, String curieText) {
```

Comment:
Once I copied an ontology, I cannot do so again. This is bad. I want to copy it into multiple fields in my measurement sheet. If I copy a second term, the previous is still highlighted (do we want this?)